Tempo/Controllers/AboutViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AboutViewController.swift
+//  Tempo
+//
+//  Created by Jesse Chen on 9/28/16.
+//  Copyright © 2016 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class AboutViewController: UIViewController {
+	@IBOutlet weak var aboutLabel: UILabel!
+	@IBOutlet weak var teamLabel: UILabel!
+    @IBOutlet weak var linksLabel: UITextView!
+	
+    let aboutText = "Cornell App Development is a project team at Cornell University dedicated to mobile app development. Every year, we take an app from idea to product, culminating with a release on the App Store. We give students the opportunity to work in a startup environment and gain practical experience in software development, design, and product management. Our team is made up of talented designers, developers, and visionaries who collaborate to bring projects to life."
+	let teamText = "Annie Cheng, Andrew Aquino, Dennis Fedorko, Ilan Filonenko, Adam ElShaer, Eileen Dai, Austin Chan, Sahil Khoja, Sara Cheong, Tiffany Zheng, Derrick Ho, Jason Wu, Mihir Chauhan, Jonathan Lee, Mark Bryan, Jesse Chen, Joseph Antonakakis, Emily Lien, Alex Zielenski, Natasha Armbrust, Daniel Li, Hong Jeon, Logan Allen, Celine Brass, Kevin Greer, Monica Ong, Amit Mizrahi, Shiv Roychowdhury, Matt Barker, Austin Astorga, Keivan Shahida, Arman Esmaili, Sanjana Kaundinya, Rishab Gupta, Ji Hun Kim"
```

Comment:
Alex isn't on the team anymore. Also, let's divide this into 3 sections (iOS, backend, and design), and order people alphabetically in these these sections.